vendors/laiier/drivers/severn_wld/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "severn_wld_actility",
```

"_" underscore is not accepted here/
```suggestion
  "name": "severn-wld",
```